### DISCUSSION TITLE
How do we need to contribute?

### DISCUSSION THREAD

**Discussion by adityalearnsgit** [codinasion-archive/.github-re#105] [General]
How do we need to contribute?

I mean do we have to raise an issue on which program we would like to work on or do we just select one from the issues on which we are interested to work in.

**Comment by harshraj8843** [comment]
Hey @adityalearnsgit 

The `good first issues` are already present in the repos.

You can comment `!assign` on any `good first issue` you want to contribute to, to be auto-assigned

```
!assign
```

---

You can also suggest new programs 👉🏻 https://github.com/codinasion/program/issues/new?assignees=&labels=add-program&template=submit-program.yml
